databases/installing-postgresql.md

```diff
@@ -19,6 +19,8 @@
 
 ### 1. Install PostgreSQL through app [On Windows].
 
+* **Reference**: [Installation Steps](https://www.guru99.com/download-install-postgresql.html)
```

Comment:
Could you please mention the website or author instead of "Installation Steps"?

Reply:
Yes I will.